packages/alert/src/close-icon.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import {Button} from '@smashing/button'
+
+const CloseButton = styled(Button)`
+  margin-left: auto;
```

Comment:
Is there any margin between text and button?

Reply:
Added 1em margin